fe2-android/app/build.gradle

```diff
@@ -186,6 +186,11 @@ copy {
     include 'examples/**'
     include 'query/**'
     include 'jsonRPC.json'
+
+    // Replace the remote part of the base URL ($id) to the local resource path
+    filter {
+        String line -> line.replaceAll("https://raw\\.githubusercontent\\.com/dedis/[a-zA-Z0-9_-]+/master/", "resource:/")
+    }
```

Comment:
This seems pretty straightforward.

It should still fix the issue.

Maybe add a link to the issue to explain the reason behind this replacement.